.github/workflows/phpunit.yml

```diff
@@ -77,16 +77,11 @@ jobs:
       - name: Remove coding standards
         run: composer remove --dev wearerequired/coding-standards --ignore-platform-reqs
 
-      - name: Install dependencies (PHP < 8.0)
-        if: ${{ matrix.php < '8.0' }}
+      - name: Install dependencies
```

Comment:
This doesn't work, see https://github.com/wearerequired/digest/pull/23/commits/ec23ce1e153995ae8f7037fa788d834085bbc508 and https://github.com/wearerequired/digest/pull/23/commits/34103806c3a774cbb14c0c8428d72b8983c9f90c. Why not keeping it as is?

Reply:
I was trying to understand the reason why it is needed. I have documented it in https://github.com/wearerequired/digest/pull/23/commits/ad4bcebfa61f20f7375b8b629e8bdd85797308cd